diff -up ./libraries/classes/Utils/HttpRequest.php.rpm ./libraries/classes/Utils/HttpRequest.php
--- ./libraries/classes/Utils/HttpRequest.php.rpm	2018-03-01 15:05:29.388126314 +0100
+++ ./libraries/classes/Utils/HttpRequest.php	2018-03-01 15:06:51.922563502 +0100
@@ -139,12 +139,12 @@ class HttpRequest
          *
          * See https://letsencrypt.org/certificates/
          */
-        $certsDir = dirname(__file__) . '/../../certs/';
+        $certsDir = '/etc/pki/tls/';
         /* See code below for logic */
         if ($ssl == CURLOPT_CAPATH) {
             $curlStatus &= curl_setopt($curlHandle, CURLOPT_CAPATH, $certsDir);
         } elseif ($ssl == CURLOPT_CAINFO) {
-            $curlStatus &= curl_setopt($curlHandle, CURLOPT_CAINFO, $certsDir . 'cacert.pem');
+            $curlStatus &= curl_setopt($curlHandle, CURLOPT_CAINFO, $certsDir . 'cert.pem');
         }
 
         $curlStatus &= curl_setopt($curlHandle, CURLOPT_RETURNTRANSFER, true);
